tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "esModuleInterop": true,
     "noEmit": true,
     // Allow spreading Maps/Sets into arrays
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Not that I really mind either way, but if `tsconfig.json` allows trailing commas then should we be using them?

Reply:
Done in c3b624b.